src/App.scss

```diff
@@ -0,0 +1,16 @@
+@import './variables';
+
+#headerImg {
+  background-image: url('burger.jpg');
+  background-size: cover;
+  height: 300px;
+  background-position: center;
+}
+
+h1 {
```

Comment:
might need a little more specificity - potential to leak out to other H1s later in the build